core/src/dird/restore.cc

```diff
@@ -564,7 +564,7 @@ void GenerateRestoreSummary(JobControlRecord* jcr,
              "  JobId:                  %d\n"
              "  Job:                    %s\n"
              "  Restore Client:         %s\n"
-             "  Start time:             %s\n"
+             "  Start time:             \"%s\" %s\n"
```

Comment:
the placeholder is misplaced: the information is displayed with `Start Time` instead of `Restore Client`